hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -292,6 +301,18 @@ private int handleMap(MemberStateImpl memberState, int count, Config config, Map
         return count;
     }
 
+    private int handleWan(MemberStateImpl memberState, int count, Map<String, LocalInstanceStats> wans,
```

Comment:
Again, signature should be changed to use `LocalWanStats` instead of `LocalInstanceStats`